custom_components/opensprinkler/sensor.py

```diff
@@ -240,6 +241,48 @@ def _get_state(self):
         return utc_from_timestamp(rdst).isoformat()
 
 
+class PauseEndTimeSensor(OpenSprinklerControllerEntity, OpenSprinklerSensor, Entity):
+    """Represent a sensor for the remaining pause time."""
+
+    def __init__(self, entry, name, controller, coordinator):
+        """Set up a new opensprinkler pause time remaining sensor."""
+        self._name = name
+        self._controller = controller
+        self._entity_type = "sensor"
+        super().__init__(entry, name, coordinator)
+
+    @property
+    def device_class(self):
+        """Return the device class."""
+        return SensorDeviceClass.TIMESTAMP
+
+    @property
+    def icon(self) -> str:
+        """Return icon."""
```

Comment:
I wonder if linting requires these comments, some of these seem pretty useless. Could help with the duplicate linting error if removed

Reply:
Another hack could be to just change the order of the properties... lol